lib/logger.js

```diff
@@ -134,40 +149,96 @@ Logger.prototype._levelToMatcher = function _levelToMatcher(level) {
     }
 };
 
+/**
+ * Parses the provided logging level for a log component,
+ * matches it with the configured specially logged components.
+ *
+ * If the message should be logged, returns an object with bunyan log level
+ * and a specialized logger for the given component, otherwise returns undefined
+ *
+ * @param {String} level a logging level
+ * @returns {Object|undefined} corresponding bunyan log level and a specialized logger
+ * @private
+ */
+Logger.prototype._getComponentLogConfig = function(level) {
+    if (level && this._log_components) {
```

Comment:
`this._log_components` will always evaluate to `true` (cf. line 20), so either drop it or use `Object.keys(this._log_components).length`